packages/did-core-test-server/suites/did-spec/utils.js

```diff
@@ -18,7 +18,14 @@ const isValidURL = (data) => {
   return valid;
 };
 
+const isValidDID = (did) => {
```

Comment:
Disagree with uppercase "DID", and "URL" in function names. Keeping it like this to go with the established code practices in this repo.

Reply:
yeah, i think the official recomendation is to use `Did` or `Url` when you are defining a function / method... obviously the ship has sailed for `new URL("https://example.com")`

Reply:
> yeah, i think the official recomendation is to use Did or Url

Would you object to a global search/replace to move to that model? I'm concerned that people might look at the test suite and then duplicate the mistake in their code.

I can do the search/replace if you don't object.